src/robot/libraries/Process.py

```diff
@@ -323,7 +323,7 @@ def __init__(self):
         self._processes = ConnectionCache('No active process.')
         self._results = {}
 
-    def run_process(self, command, *arguments, **configuration):
+    def run_process(self, command, log_level=None, *arguments, **configuration):
```

Comment:
`log_level` cannot be a positional argument because it would break pretty much all current usages of this keyword. It should be a keyword-only argument, but I'm not sure is it better to be a separate argument or should it be passed in in `**configuration` like other such options. The reason we've used `**configuration` instead of real keyword-only arguments is that this code was written originally for Python 2 that didn't even have keyword-only arguments. It probably should be refactored now, but that should be done separately from this PR. In the current situation including it in `**configuration` is probably easiest and most consistent.

Reply:
I am not sure, if it is properly done  by just adding `log_level` through `ProcessConfiguration` `__init__`.